src/mapgen_valleys.cpp

```diff
@@ -897,101 +783,150 @@ void MapgenValleys::dustTopNodes()
 }
 
 
-void MapgenValleys::generateSimpleCaves(s16 max_stone_y)
+void MapgenValleys::generateCaves(s16 max_stone_y)
```

Comment:
We'll consider but i'm not keen, maybe later, this is a specialised mapgen and most of the tunnel stuff is 3D noise tunnels which uses the mapgen noises. In other mapgens the 3D noise tunnels are kept internal while the pseudorandom caves are in cavegen.cpp.